torchvision/transforms/functional_pil.py

```diff
@@ -474,7 +474,7 @@ def _parse_fill(fill, img, min_pil_version, name="fillcolor"):
 
 
 @torch.jit.unused
-def affine(img, matrix, resample=0, fillcolor=None):
+def affine(img, matrix, interpolation=0, fill=None):
```

Comment:
I assume that JIT will complain if you change the numeric default value with the enum, correct?

Reply:
This is private PIL affine method and in addition torchscript should not pass here.

Reply:
I suppose we could move the definitions to avoid circular dependencies but I understand that this PR has been dragging for weeks now, so I don't think we should block it further for this.

Reply:
IMO, the most suitable option is to create another module with modes, such that `F`, `F_t` and `F_pil` could import it and reuse if needed. If there is an argumented need to do that, I'm happy to recoded no matter of how many time the PR is open... IMO, it is better to properly design things and go slower that faster and have a poor API :)

Reply:
I think your proposed changes on the transforms API are very elegant. Your choice of using enums instead of raw values has multiple benefits (less mistakes, clearer API, easier to search usages etc), so as a user I would definitely love it to be similar across all functional APIs. Again I would not say this is a deal breaker for me but more of a nice-to-have.